sdk/keyvault/azure-security-keyvault-jca/pom.xml

```diff
@@ -191,6 +191,12 @@
           <version>3.6.28</version> <!-- {x-version-update;org.mockito:mockito-core;external_dependency} -->
           <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>com.azure</groupId>
+          <artifactId>azure-core-test</artifactId>
+          <version>1.6.2</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
+          <scope>test</scope>
+        </dependency>
```

Comment:
Where do we use it?

Reply:
This library is used to show our test info in the pipeline, you could check [here](https://dev.azure.com/azure-sdk/public/_build/results?buildId=915354&view=logs&j=bb4251f5-b8e9-5761-71ac-1cf2c72c9b27&t=231b82bd-ed1a-5f58-da3e-bace0d3d8d96&l=307), or there wouldn't be any logs that if our tests are scheduled or not. Same for the live test pipeline, do you think whether we should keep this dependency?